manifests/m/Microsoft/AzureCLI/2.42.0/Microsoft.AzureCLI.yaml

```diff
@@ -0,0 +1,16 @@
+PackageIdentifier: Microsoft.AzureCLI
+PackageVersion: 2.42.0
+PackageName: Microsoft Azure CLI
+Publisher: Microsoft Corporation
+Moniker: azure-cli
+PackageUrl: https://docs.microsoft.com/en-us/cli/azure/
+License: Copyright (C) Microsoft Corporation
+ShortDescription: The Azure command-line interface (Azure CLI) is a set of commands used to create and manage Azure resources. The Azure CLI is available across Azure services and is designed to get you working quickly with Azure, with an emphasis on automation.
+Installers:
+- Architecture: x64
+  InstallerUrl: https://azcliprod.azureedge.net/msi/azure-cli-2.42.0.msi
+  InstallerType: msi
+  InstallerSha256: b8ce13727e044efdf350e4d0bd49ce2aee418fcc6a1c10fe52ad046a5c97b521
+PackageLocale: en-US
+ManifestType: singleton
+ManifestVersion: 1.0.0
```

Comment:
Please move to Manifest version 1.2.0

Reply:
@Trenly,
Please help close this PR, we will re-issue a PR to cover this PR.
Is there any relevant doc to guide how to upgrade to Manifest version 1.2.0?
Can we upgrade to Manifest version 1.2.0 in the next version?

Reply:
@Trenly,
Actually, this file is hardcoded, can we simply modify ManifestVersion from 1.0.0 to 1.2.0?

Reply:
> @Trenly,
> Actually, this file is hardcoded, can we simply modify ManifestVersion from 1.0.0 to 1.2.0?

Yes, that should work